### DISCUSSION TITLE
onEmailPasswordSignUp not implemented yet?

### DISCUSSION THREAD

**Discussion by perevezencev** [edgedb/edgedb-js#1066] [Q&A]
Hi all! My app has email and password signup. So I need to use `onEmailPasswordSignUp` but it is not called

After a little research, I did not find an implementation of this method. Is this not supported now?

<img width="859" alt="image" src="https://github.com/user-attachments/assets/da976252-59fc-4152-8546-1eab699c762c">

**Comment by scotttrinh** [comment]
It's used here: https://github.com/edgedb/edgedb-js/blob/master/packages/auth-nextjs/src/shared.ts#L596-L597

Are you using Server Actions or are you calling this route from a form or fetch call?

**Reply by perevezencev** [reply]
I am using Server Actions
